src/Components/FontIcon.re

```diff
@@ -8,28 +8,22 @@ open Revery.UI;
 open Oni_Core.CamomileBundled.Camomile;
 module ZedBundled = Oni_Core.ZedBundled;
 
+module Styles = {
+  open Style;
+
+  let text = (~fontFamily, ~fontSize, ~color) => [
+    Style.fontFamily(fontFamily),
+    Style.fontSize(fontSize),
+    Style.color(color),
+    textWrap(Revery.TextWrapping.NoWrap),
+  ];
+};
+
 let codeToIcon = icon => ZedBundled.singleton(UChar.of_int(icon));
 
 let make =
-    (
-      ~icon,
-      ~fontFamily=FontAwesome.fontFamily,
-      ~fontSize=15.,
-      ~backgroundColor,
-      ~color,
-      ~margin=0,
-      (),
-    ) =>
+    (~icon, ~fontFamily=FontAwesome.fontFamily, ~fontSize=15., ~color, ()) =>
   <Text
     text={codeToIcon(icon)}
-    style=[
-      Style.fontFamily(fontFamily),
-      Style.fontSize(fontSize),
-      Style.color(color),
-      Style.backgroundColor(backgroundColor),
-      Style.margin(margin),
-      Style.height(fontSize |> int_of_float),
-      Style.width(fontSize |> int_of_float),
```

Comment:
Removed these, `backgroundColor` is no longer needed, `width` and `height` messed with centering as it wasn't possible to control the alignment of the icon within this box, and `margin` doesn't really belong here (any more than any other style at least) and can be added when needed just by wrapping it in another view.

Reply:
Glad to see these go away!